RF24Gateway.h

```diff
@@ -223,6 +223,11 @@ class ESBGateway
      */
     uint8_t routingTableSize;
 
+    /**
+     * How many times the network has returned error 160 NETWORK_OVERRUN
```

Comment:
```suggestion
     * How many times the network has returned error 160 NETWORK_OVERRUN.
     * The radio's RX FIFO was flushed each time, to recover from the flood of incoming data.
```
Just so people don't have to look up what `NETWORK_OVERRUN` means. Sadly doxygen doesn't easily cross reference to other doxygen docs. Trust me, I've tried.

Reply:
I'm not sure what to put in the docs now. My next proposed change for RF24Network won't directly flush the buffer if this happens, it will just break out of the `available()` loop so the network can transmit as required. I guess that will cause the RX buffers to get flushed anyway?

Reply:
I'm curious what the FIFO looks like when dynamic payload size is invalid. I thought the only way to fix that was to flush the RX FIFO.

Reply:
Oh, I'm confusing causes of this error.  We can modify the doc later when solution is stablized.

Reply:
> I'm curious what the FIFO looks like when dynamic payload size is invalid. I thought the only way to fix that was to flush the RX FIFO.

If you look at my PR
https://github.com/nRF24/RF24Network/pull/250
I added a new type for detecting the corruption, ex: when RF24 returns 0 for dynamic payload length.
Do you mean like attempt a 32-byte read when this happens and see what the data looks like?

Reply:
I also want to ammend this PR to also display number for how many times corruption has happened, along with any network overruns.

Reply:
> Do you mean like attempt a 32-byte read when this happens and see what the data looks like?

Not in production code. I really just wanted to see if the RX FIFO is corrupted or just the dynamic payload size data was corrupted. What I'm trying to get at is if the RX FIFO can be "rescued" when the reported dynamic size is invalid. In my head, I'm imagining:
```cpp
uint8_t RF24::getDynamicPayloadSize(void)
{
    uint8_t result = read_register(R_RX_PL_WID);

    if (result > 32 || !result) {
        // double check the radio intended to report
        // the bad result we just got
        result = read_register(R_RX_PL_WID);
        if (result > 32 || !result) {
            flush_rx();
            return 0;
        }
    }
    return result;
}
```
This idea doesn't bode well for compile size of apps that use dynamic payload size...

Reply:
Reading any bytes from the FIFO (w/ dynamic payloads on) may (or may not) end up clearing a payload from the FIFO.

Reply:
> Reading any bytes from the FIFO (w/ dynamic payloads on) may (or may not) end up clearing a payload from the FIFO.

In this case when R_RX_PL_WID returns 0 it won't. 

I've tested that with the default behaviour of RF24Network right before I put in the RF24 Dynamic Payloads fix.
 
Available being stuck in the endless return true loop while read(&data, 1); (or any length) has no effect on the FIFO.